src/main/java/net/sf/jabref/logic/util/OS.java

```diff
@@ -11,4 +11,24 @@
     public static final boolean LINUX = osName.startsWith("linux");
     public static final boolean WINDOWS = osName.startsWith("win");
     public static final boolean OS_X = osName.startsWith("mac");
+    
+    public static final String guessProgramPath(String programName, String windowsDirectory) {
+        if (OS.WINDOWS) {
+            String progFiles = System.getenv("ProgramFiles(x86)");
```

Comment:
According to Stack Overflow use have to use `System.getenv("%programfiles% (x86)"); ` instead. http://stackoverflow.com/questions/4851359/java-get-program-files-path